Extension/package.json

```diff
@@ -42,6 +42,153 @@
+            "when": "isMac",
+            "media": {
+              "altText": "A markdown file with instructions for installing Clang on macOS.",
+              "path": "walkthrough/install-clang-macos.md"
+            }
+          },
+          {
+            "id": "verify.compiler.linux",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your system to configure IntelliSense for your project. \n[Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isLinux",
+            "media": {
+              "altText": "A markdown file with instructions for installing GCC on Linux.",
+              "path": "walkthrough/install-gcc-linux.md"
+            }
+          },
+          {
+            "id": "verify.compiler.windows",
+            "title": "Install a C++ compiler",
+            "description": "The C++ extension uses the C++ compiler on your system to configure IntelliSense for your project. \n[Check for C++ compilers](command:C_Cpp.CheckForCompiler)",
+            "when": "isWindows",
+            "media": {
+              "altText": "A markdown file with instructions for installing MSVC on Windows.",
+              "path": "walkthrough/install-compiler-windows.md"
+            }
+          },
+          {
+            "id": "open.project",
+            "title": "Open your project folder",
+            "description": "Open an existing folder that contains your C++ project, or create a new folder. \n[Pick a folder](command:workbench.action.files.openFolder)",
+            "when": "workspaceFolderCount == 0",
+            "media": {
+              "altText": "A screenshot that shows a C++ project opened in the file Explorer.",
+              "path": "walkthrough/cpp-folder-explorer.png"
+            }
+          },
+          {
+            "id": "setup.intellisense",
+            "title": "Configure C++ IntelliSense",
+            "description": "Verify important IntelliSense settings like Compiler Path, Compiler Args, IntelliSense Mode, and Include Path. \n[Open IntelliSense Configuration](command:C_Cpp.ConfigurationEditUI?-2)",
+            "media": {
+              "altText": "A screenshot that shows the IntelliSense Configurations UI with important settings highlighted.",
+              "path": {
+                "light": "walkthrough/intellisense-config-light.png",
+                "dark": "walkthrough/intellisense-config-dark.png",
+                "hc": "walkthrough/intellisense-config-hc.png"
+              }
+            }
+          },
+          {
+            "id": "build.project.mac",
+            "title": "Build your C++ file",
+            "description": "Create a default build task for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-clang-mac#_build-helloworldcpp)",
+            "when": "isMac",
+            "media": {
```

Comment:
This reference to "Mac" should be "macOS" to be consistent with the other uses.

Reply:
fixed